doc/api/util.md

```diff
@@ -1392,11 +1392,10 @@ added: v10.0.0
 * Returns: {boolean}
 
 Returns `true` if the value is a native `External` value.
-A native `External` value is a special type of object whose
-data is not stored within the JavaScript managed heap
-and does not conform to standard JavaScript types. Such
-objects are created either by Node.js internals or native
-addons and are wrapped by a JavaScript object.
+
+A native `External` value is a special type of object that contains a
+C++ `void*` pointer for access from native code, and has no other properties.
+Such objects are created either by Node.js internals or native addons.
```

Comment:
@addaleax - while this is more specific definition of `external` types, the existing text is more consumable by end users IMO. For easy consumption while being accurate, shall I suggest to retain `data is not stored within the JavaScript managed heap and does not conform to standard JavaScript types..` phrase?

Reply:
@gireeshpunathil Sorry, but I have to disagree. The previous text was far from ideal, imo:

1. `External`s are not concerned with where data is stored, or whether data is associated with the `External` at all. They could even be used to store a single machine-word-sized value (which would mean that the `External`’s data is actually fully stored on the JS heap).
 2. Saying that an object “does not conform to standard JavaScript types” without describing in what way it doesn’t conform is just not helpful, and worse, `External` objects *do* conform to stanrdard JS behaviour – in JS, they’re indistinguishable from `Object.freeze(Object.create(null))`.

I’m happy to take updates here, but I’d like to keep things helpful and accurate.

Reply:
```suggestion
Such objects are created either by Node.js internals or native addons.
In JS, they’re indistinguishable from `Object.freeze(Object.create(null))`.
```

Would it be possible to reword the ``...contains a C++ `void*` pointer`` part? A pure JS developers might not be able to follow the part otherwise.

Reply:
@BridgeAR I’ve reworded the text a bit along the lines of your suggestions, but I’m not quite sure what you had in mind for the second one?

Reply:
I did not think of anything specific. I guess it's fine as it is.